.circleci/deploy-linux.sh

```diff
@@ -44,18 +50,17 @@ for PYBIN in $PYBINS; do
   # Need to export the current bin path so that scikit-build can find the pip
   # installed cmake binary.
   export PATH=${PYBIN}:${PATH}
-  "${PYBIN}/python" -m pip install cython scikit-build cmake --ignore-installed -q --progress-bar=off
-  rm -rf numpy-1.14.6
-  curl -sSLO https://github.com/numpy/numpy/archive/v1.14.6.tar.gz
-  tar -xzf v1.14.6.tar.gz
-  cd numpy-1.14.6
-  rm -f numpy/random/mtrand/mtrand.c
-  rm -f PKG-INFO
-  "${PYBIN}/python" -m pip install . --no-deps --ignore-installed -v --progress-bar=off -q
+  "${PYBIN}/python" -m pip install cython scikit-build cmake oldest-supported-numpy --ignore-installed -q --progress-bar=off
   "${PYBIN}/pip" wheel ~/ci/freud/ -w ~/wheelhouse/ --no-deps --no-build-isolation --no-use-pep517
 done
 
-# Update RPath for wheels
+# Install patched auditwheel (fixes RPATHs for libfreud/libtbb, issue #136).
+cd ~/
+git clone https://github.com/mayeut/auditwheel.git -b issue136 auditwheel
```

Comment:
@joaander This branch was merged and deleted. We can install from the upstream's `master` branch now.
```suggestion
git clone https://github.com/pypa/auditwheel.git -b master auditwheel
```